compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -255,11 +256,14 @@
     private DiagnosticCode diagCode;
     private BType resType;
     private Unifier unifier;
+    private PackageID rootPackage;
 
     private Map<BVarSymbol, BType.NarrowedTypes> narrowedTypeInfo;
     // Stack holding the fall-back environments. fall-back env is the env to go back
     // after visiting the current env.
     private Stack<SymbolEnv> prevEnvs = new Stack<>();
+    private Set<String> configKeys = new HashSet<>();
```

Comment:
Let's move this to the method local variable since sub modules with unimported modules can cause issues.